config/crd/kustomization.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.1
+- github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v0.5.0-rc1
+- https://raw.githubusercontent.com/kubernetes-sigs/gateway-api/v0.5.0-rc1/config/crd/experimental/gateway.networking.k8s.io_referencepolicies.yaml
```

Comment:
Needed to be installed explicitly because kustomize seems to omit the ReferencePolicies CRD by default due to it being marked `deprecated: true`. Remote resource install refs https://github.com/kubernetes-sigs/kustomize/issues/970#issuecomment-909419318